### ISSUE TITLE
(Re)publish the library on maven central

### ISSUE THREAD

**Issue by ilya-g** [Kotlin/kotlinx.collections.immutable#96]
Considering the news that jcenter is going to sunset soon, we need to setup publishing to maven central repository.

**Comment by Jose-Parente** [comment]
The latest canary of Android Studio has now removed jcenter() in favor of mavenCentral()

**Comment by whyrising** [comment]
+1

**Comment by WebTiger89** [comment]
+1

**Comment by laurenyew-nytimes** [comment]
+1

**Comment by ykws** [comment]
https://github.com/Kotlin/kotlinx.html/issues/173#issuecomment-800504909

Is it recommended to use the following snippet similar to the linked issue?

```
jcenter {
    content {
        includeModule("org.jetbrains.kotlinx", "kotlinx-collections-immutable-jvm")
    }
}
```

**Comment by saurabhraj042** [comment]
facing same issue

**Comment by jeremyrempel** [comment]
> [Kotlin/kotlinx.html#173 (comment)](https://github.com/Kotlin/kotlinx.html/issues/173#issuecomment-800504909)
> 
> Is it recommended to use the following snippet similar to the linked issue?
> 
> ```
> jcenter {
>     content {
>         includeModule("org.jetbrains.kotlinx", "kotlinx-collections-immutable-jvm")
>     }
> }
> ```

Yes. The snippet just means the only dependency that can be obtained from jcenter is the specified one.

**Comment by boswelja** [comment]
Seems to be live now
https://mvnrepository.com/artifact/org.jetbrains.kotlinx/kotlinx-collections-immutable-jvm/0.3.3
I've removed jcenter from my project and I haven't run into any issues yet

**Comment by maxcruz** [comment]
Unfortunately, it's still failing when you compile without jcenter():

``` 
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.3.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/org/jetbrains/kotlinx/kotlinx-collections-immutable-jvm/0.3.3/kotlinx-collections-immutable-jvm-0.3.3.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/kotlinx/kotlinx-collections-immutable-jvm/0.3.3/kotlinx-collections-immutable-jvm-0.3.3.pom
     Required by:
         project :app > androidx.compose.ui:ui:1.0.0-beta02 > androidx.compose.runtime:runtime:1.0.0-beta02

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

```

**Comment by qurbonzoda** [comment]
The new version of the library, `0.3.4`, is published to the Maven Central repository.
Thank you for the patience!
Readme will be updated soon.

**Comment by tajchert** [comment]
I'm also facing the same issue, can you also republish `0.3.3` version?

**Comment by maxcruz** [comment]
> I'm also facing the same issue, can you also republish `0.3.3` version?
> 
> Current version of `androidx.compose.runtime:runtime:1.0.0-beta03` on Android uses `0.3.3` version:
> 
> ```
> Execution failed for task ':xxxx'.
> > Could not resolve all files for configuration ':xxxx:debugAndroidTestRuntimeClasspath'.
>    > Could not resolve org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.3.
>      Required by:
>          project :xxx > androidx.compose.runtime:runtime:1.0.0-beta03
>       > Could not resolve org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.3.
> ```

 I guess this issue is going to be solved when compose-ui moves to 1.0.0-beta04
As a workaround, you could add version 0.3.4 explicitly to force the dependency resolution. 

`implementation "org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.4"`

**Comment by qurbonzoda** [comment]
As of now we have no plans to republish the old versions to Maven Central.
We have discussed the issue with compose-ui folks, they will shade the dependency or will move to 0.3.4.
